map.json

```diff
@@ -230,6 +236,12 @@
          "imagewidth":320,
          "margin":0,
          "name":"tileset6_export",
+         "properties":[
+          {
+           "name":"tilesetCopyright",
+           "type":"string",
+           "value":"https:\/\/workadventu.re\/"
```

Comment:
We should probably have 2 properties:

"copyright": "© 2020 Full Name <email>"
"license": "CC BY-SA 3.0"

Reply:
Ok, this is a new task.

Reply:
At least, put:

"© 2021 WorkAdventure - Lincense: WORKADVENTURE SPECIFIC RESOURCES LICENSE (see LICENCE.assets file)"

Reply:
Good catch